includes/managers/icons.php

```diff
@@ -516,6 +517,29 @@ public function register_admin_tools_settings( Tools $settings ) {
 		] );
 	}
 
+	/**
+	 * Get redirect URL when upgrading font awesome.
+	 *
+	 * @return string
+	 */
+	public function get_upgrade_redirect_url() {
+		// PHPCS - No need to verify nonce, the 'redirect_to' could be only post id, and if it is not the
```

Comment:
Why not adding a nonce?

Reply:
Seems like there is no need for the nonce, only the ID is transferred through the GET params.

Reply:
It may be changed over time.

Reply:
Done